src/client/pages/PageRoutes/hooks/useRoutes.tsx

```diff
@@ -67,7 +67,10 @@ export const useRoutes = () => {
             {/* Login */}
             <Route path={Routes.Login.path.relative} element={<Login />}>
               <Route index element={<LoginForm />} />
-              <Route path={Routes.LoginInvitation.path.relative} element={<LoginInvitation />} />
+              <Route path={Routes.LoginInvitation.path.relative}>
+                <Route index element={<LoginInvitation />} />
+                <Route path={Routes.LoginInvitationLocal.path.relative} element={<LoginInvitationLocal />} />
```

Comment:
I changed the routes to this and now I can use Outlet:
```
<Route path={Routes.Login.path.relative} element={<Login />}>
    <Route index element={<LoginForm />} />
    <Route path={Routes.LoginInvitation.path.relative} element={<LoginInvitation />}>
      <Route path={Routes.LoginInvitationLocal.path.relative} element={<LoginInvitationLocal />} />
    </Route>
    <Route path={Routes.LoginResetPassword.path.relative} element={<LoginResetPassword />} />
</Route>
```
However, to be able to keep the Buttons in LoginInvitation, I need to get this function from LoginInvitationLocal:
`const onAcceptInvitation = useOnAcceptInvitationLocal({ formData, invitationUuid, setErrors, showPassword2 })` in LoginInvitation, that way if I am not in Local, the button will take me there and otherwise it will trigger accepting the invitation.

To get that function I tried passing a ref through the Outlet so that the the LoginInvitationLocal can update it:
```
export type InvitationContext = {
  onAcceptInvitationLocalRef: React.MutableRefObject<() => void>
}
const Invitation: React.FC = () => {
...
  const onAcceptInvitationLocalRef = useRef<() => void>()
  const context: InvitationContext = { onAcceptInvitationLocalRef }
 ...
 return(
...
<Outlet context={context} />
...
)
```
This did not work as I couldn't trigger a re render to the parent Invitation from InvitationLocal after updating the function:
```
const InvitationLocal: React.FC = () => {
  ...
  const { onAcceptInvitationLocalRef } = useOutletContext<InvitationContext>()
  onAcceptInvitationLocalRef.current  = useOnAcceptInvitationLocal({ formData, invitationUuid, setErrors, showPassword2 })
...
```

I would appreciate any suggestions on how to handle this. 
The other option I see is creating a slice with the form state and dispatching actions, but I think that would need a new PR.

Reply:
> if in LoginInvitation -> navigate to LoginInvitationLocal
> if in LoginInvitationLocal -> submit form.

To submit the form from the parent route LoginInvitation, I would need to access the `formData` from the child route LoginInvitationLocal. To do that while using Outlet, I would need to use context API and wrap both routes. Or, I could lift the form state up, but that would leave us with the same code as before 😅 

If context API is okay for you, I can implement it.

Reply:
Added Outlet to the most recent commit, but with conditional button render so we can take it from there 👍

Reply:
Sure. Should I add it to the existing `store/login` or create a new slice in `store/ui/...`? since it is just form data

Reply:
Added. Now the button is only present in Invitation and InvitationLocal only handles the form.